docs/products/postgresql/howto/prevent-full-disk.rst

```diff
@@ -0,0 +1,58 @@
+Prevent PostgreSQL full disk issues
+===================================
+
+If your Aiven for PostgreSQL service runs out of disk space, the service will start malfunctioning which will preclude the proper backups creation. 
```

Comment:
If you are keeping "precluding", then I think it needs to be
```
will start malfunctioning, which will preclude proper backup creation.
```
(so removing `the` and `s`), which is perfectly OK, or

Reply:
Thanks fixed! Could you please do one more overall check and then proceed with the usual process?